package.json

```diff
@@ -1,4 +1,52 @@
 {
```

Comment:
I'm not able to run `npm install` properly with this file, looks like there are some errors here. It doesn't look like we're leveraging any new libraries here, so we can probably just restore this to what it is upstream.